packages/manager/modules/manager-pci-common/package.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "@ovh-ux/manager-pci-common",
+  "version": "0.0.0",
+  "private": true,
+  "sideEffects": false,
+  "type": "module",
+  "main": "./src/index.ts",
+  "scripts": {
+    "coverage": "vitest run --coverage",
+    "dev": "tsc",
+    "lint": "eslint ./src",
+    "test": "vitest run"
+  },
+  "dependencies": {
+    "clsx": "2.1.1",
+    "date-fns": "^3.3.1",
+    "element-internals-polyfill": "^1.3.10",
+    "file-saver": "^2.0.5",
+    "i18next": "^23.8.2",
+    "i18next-http-backend": "^2.4.3",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-hook-form": "^7.50.1",
+    "react-i18next": "^14.0.5",
+    "react-router-dom": "^6.22.0",
+    "react-use": "^17.5.0"
```

Since they are already defined in `peerDependencies`, you can remove it from here.
```suggestion
    "react-use": "^17.5.0"
```